src/Mono.Android/metadata

```diff
@@ -1381,6 +1381,25 @@
   <attr path="/api/package[@name='android.media']/interface[@name='MediaCas.EventListener']" name="argsType">MediaCasEventArgs</attr>
   <attr path="/api/package[@name='android.media']/interface[@name='MediaCas.EventListener']/method[@name='onEvent']/parameter[@name='MediaCas']" name="managedName">mediaCas</attr>
 
+  <!-- Until API Level 26, android.graphics.Color was static. In API Level 26,
+  it became instantiable and got instance members.
+
+  Xamarin.Android had wrong premise that android.graphics.Color can be totally
+  different thing and thus introduced completely different API with the same
+  type name. Wrong wrong wrong. It was wrong decision made back in 2010.
+
+  We will have to enable this line and bring back some sanity.
+  -->
+  <!-- attr path="/api/package[@name='android.graphics']/class[@name='Color']" name="managedName">AndroidColor</attr -->
```

Comment:
Why is this line commented out? Why not enable it now?

Reply:
This causes generator generating broken code because it depends on "_Color" type!